lib/util.js

```diff
@@ -517,7 +517,7 @@ function formatCollectionIterator(ctx, value, recurseTimes, visibleKeys, keys) {
   var nextRecurseTimes = recurseTimes === null ? null : recurseTimes - 1;
   var vals = mirror.preview();
   var output = [];
-  for (let o of vals) {
+  for (const o of vals) {
```

Comment:
You are correct. Looks like it doesn't. I tried

```js
for (const a of [1.3, 2, 3, 4]) {
  console.log(a);
}
```
and it prints only `1.3`. Looks like the rule is buggy.